examples/04_glm_first_level/plot_first_level_details.py

```diff
@@ -189,38 +215,55 @@ def make_localizer_contrasts(design_matrix):
 # As this script will be repeated several times, we encapsulate model
 # fitting and plotting in a function that we call when needed.
 #
-from nilearn import plotting
+from nilearn.plotting import plot_stat_map
 
 
 def plot_contrast(first_level_model):
     """Specify, estimate and plot the main contrasts \
         for given a first model.
     """
+    # Set vmax to keep range of values equal
+    # across models / contrasts
+    # vmax = 10
+
     design_matrix = first_level_model.design_matrices_[0]
+
     # Call the contrast specification within the function
     contrasts = make_localizer_contrasts(design_matrix)
-    plt.figure(figsize=(20, 4))
+    plt.figure(figsize=(10, 10))
+
+    # To make sure we plot the same slice across models.
+    cut_coords = {
+        "left - right button press": 66,
+        "audio - visual": -3,
+        "computation - sentences": -6,
+        "horizontal-vertical": 24,
+    }
+
     # compute the per-contrast z-map
     for index, (contrast_id, contrast_val) in enumerate(contrasts.items()):
-        ax = plt.subplot(1, len(contrasts), 1 + index)
-        z_map = first_level_model.compute_contrast(
-            contrast_val, output_type="z_score"
+        ax = plt.subplot(2, 2, 1 + index)
+        results = first_level_model.compute_contrast(
+            contrast_val, output_type="all"
         )
-        plotting.plot_stat_map(
-            z_map,
+        plot_stat_map(
+            results["stat"],
             display_mode="z",
-            threshold=3.0,
+            threshold=1,
```

Comment:
Why do you change the threshold to 1 ?

Reply:
because these plots are now on the stats estimates not Z maps, so the rule of thumbs of 1.96 and 3 don't apply

the transparency uses the z maps

Reply:
OK, but then make it explicit. Otherwise, people will be surprised.